helix/devops/development/setting-up.rst

```diff
@@ -1,26 +1,16 @@
 Setting up a development environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Optimising the task of setting up a local development environment is not
-just about on-boarding new developers or resetting a developer’s local
-environment – although these tasks can be important to optimise. It is
-also – and sometimes more importantly – about being able to consistently
-and quickly set up a given version of the complete clean implementation
-for testing and troubleshooting.
-
-Typically, after the initial build phase – where there often is an
-abundance of resources – comes the support or extension phase. In this
-phase, where there are often fewer resources and maybe even less
-experience with the implementation and processes, there is often a need
-for working on multiple branches at the same time, testing on
-production-like environments, the need to establish troubleshooting
-environments, etc. and so the need to quickly spin up a specific version
-of the local development environment increases.
-
-Setting up a local developer machine with a running environment can be
-more or less complex given business requirements and dependencies, but
-the following is an example of the tasks that could be involved and
-example of what each might contain:
+Optimising the task of setting up a local development environment is
+a critical optimization point for on-boarding new developers
+or resetting a developer’s local environment. It is also
```

Comment:
Multiple locales used here -- "Optimize" vs "Optimise". Should select one or the other. Also confusing sentence. Suggest removing the first "Optimising".

Reply:
Fixed